ScrabbleSinatra.rb

```diff
@@ -0,0 +1,49 @@
+require "sinatra"
+require "sinatra/reloader"
+require "./lib/Scrabble"
+
+class ScrabbleSinatra < Sinatra::Base
+  register Sinatra::Reloader
+
+  get "/" do
+    erb :home
+  end
+
+  get "/score" do
+    erb :score
+  end
+
+  post "/score" do
+    @word = params[:word]
+    # Using instance variable because refering to @word on score.erb.
+    # If local variable, score.erb is outside the scope.
+    @score = Scrabble::Scrabble.score(@word)
+    @letters = Scrabble::Scrabble.letter_array(@word)
+    @letter_scoring = Scrabble::Scrabble.letter_scoring(@letters)
+    erb :score
+
+  end
+
+  get "/multi_score" do
+    erb :multi_score
+  end
+
+  post "/multi_score" do
```

Comment:
I wonder if you could combine all of this logic into one method in your scrabble class so that the logic in the controller method could just retrieve an array of data that will be handled based on the word.